Formula/mesos.rb

```diff
@@ -86,6 +54,13 @@ def install
 
     ENV.cxx11
 
+    # macOS no longer provides JavaVM.framework. Fix broken configure script
+    # that assumes this framework exists for JNI linking.
+    ENV["JAVA_CPPFLAGS"] = "-I#{Formula["openjdk@8"].include}"
+    ENV["JAVA_HOME"] = Language::Java.java_home("1.8")
+    ENV["JAVA_TEST_LDFLAGS"] = "-L#{ENV["JAVA_HOME"]}/lib/server -ljvm"
```

Comment:
```suggestion
    ENV["JAVA_TEST_LDFLAGS"] = "-L#{ENV["JAVA_HOME"]}/jre/lib/server -ljvm"
```

Reply:
Good catch, thanks! Stupid Java 8 not matching later Java versions...